src/Microsoft.VisualStudio.Services.Agent/Util/NetFrameworkUtil.cs

```diff
@@ -154,9 +154,13 @@ private void InitVersions()
                             {
                                 versionString = "4.6.1";
                             }
-                            else if (release > 394271)
+                            else if (release > 394271 && release <= 460797)
```

Comment:
this doesn't line up https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed